tests/integration/components/frost-fixed-table-test.js

```diff
@@ -263,8 +266,15 @@ describe(test.label, function () {
 
         heroes.forEach((hero, rowIndex) => {
           middleColumns.forEach((column, columnIndex) => {
-            it(`should set a hook on the cell in row: ${rowIndex}, column: ${columnIndex}`, function () {
-              const $cell = $hook('myTable-middle-cell', {row: rowIndex, column: columnIndex})
+            // The fixed table uses column indices that are unique across left/middle/right,
+            // so we expect to find hooks and event index identifiers that start with an index
+            // that is offset by the number of columns in the sections to the left of us.
+            const colIndex = columnIndex + 1
```

Comment:
Maybe `globalIndex` instead of `colIndex`? Since it's not exactly intuitive why `col` would be a different index than `column`.